airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination_async/DetectStreamToFlush.java

```diff
@@ -203,14 +203,15 @@ long estimateSizeOfRunningWorkers(final StreamDescriptor stream, final long curr
    * resources optimally. Then get rid of old stuff (time to value for the user and, generally, as the
    * age of the last record grows, the likelihood of getting any more records from that stream
    * decreases, so by flushing them, we can totally complete that stream). Finally, tertiary sort by
-   * name so the order is deterministic.
+   * name so the order is deterministic. Ignore queues with 0 bytes i.e. no records.
    *
    * @param streams streams to sort.
    * @return streams sorted by priority.
    */
   @VisibleForTesting
   List<StreamDescriptor> orderStreamsByPriority(final Set<StreamDescriptor> streams) {
```

Comment:
nit: rename to `getFlushableStreamsSortedByPriority`? to clarify that it excludes some streams